src/pages/html/images/index.md

```diff
@@ -8,6 +8,28 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A simple HTML Image element can be included in an HTML document like this:
+
+```
+<img src="path/to/image/file" alt="this is a cool picture>
+```
+
+```alt``` tags provide alternate text for an image. One use of the ```alt``` tag is for visually impaired people using a screen reader; they can be read the ```alt``` tag of the image in order to understand the image's meaning.
+
+Note that the path to the image file can be either relative or absolute. Also, remember that the ```img``` element is self-closing, meaning that it does not need a closing ```</img>``` tag and instead closes with just a single ```<```.
```

Comment:
\`img`

this supposes to be \`<img \/>`.

\`<`

Reply:
Thanks! Should I just edit my pull request...?